tests/e2e/test_rosacli_idp.go

```diff
@@ -680,4 +680,162 @@ var _ = Describe("Edit IDP",
+
+						if profile.ClusterConfig.ExternalAuthConfig {
+							Skip("IDP is not supported for external auth")
+						}
+
+						By("Create admin")
+						output, err := rosaClient.User.CreateAdmin(clusterID)
+						Expect(err).To(BeNil())
+						textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+						Expect(textData).Should(ContainSubstring("Admin account has been added"))
+
+						By("Create one htpasswd idp with --users flag")
+						output, err = idpService.CreateIDP(clusterID, idpNames[0],
+							"--type", idpType,
+							"--users", usersValue,
+							"-y")
+						Expect(err).To(BeNil())
+						textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+						Expect(textData).Should(ContainSubstring("Identity Provider '%s' has been created", idpNames[0]))
+
+						By("Create one htpasswd idp with --form-file")
+						multipleuserPasswd, err := common.GenerateMultipleHtpasswdPairs(3)
+						Expect(err).To(BeNil())
+						location, err := common.CreateTempFileWithPrefixAndContent("htpasswdfile", strings.Join(multipleuserPasswd, "\n"))
+						Expect(err).To(BeNil())
+						defer os.RemoveAll(location)
+						output, err = idpService.CreateIDP(
+							clusterID, idpNames[1],
+							"--type", idpType,
+							"--from-file", location,
+							"-y")
+						Expect(err).To(BeNil())
+						textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+						Expect(textData).Should(ContainSubstring("Identity Provider '%s' has been created", idpNames[1]))
+						Expect(textData).Should(ContainSubstring("To log in to the console, open"))
+						Expect(textData).Should(ContainSubstring("and click on '%s'", idpNames[1]))
+
+						By("List IDPs")
+						idpTab, _, err := idpService.ListIDP(clusterID)
+						Expect(err).To(BeNil())
+						Expect(idpTab.IsExist("cluster-admin")).To(BeTrue())
+						Expect(idpTab.IsExist(idpNames[0])).To(BeTrue())
+						Expect(idpTab.IsExist(idpNames[1])).To(BeTrue())
+
+						By("Delete admin")
+						output, err = rosaClient.User.DeleteAdmin(clusterID)
+						Expect(err).To(BeNil())
+						textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+						Expect(textData).Should(ContainSubstring("Admin user 'cluster-admin' has been deleted"))
+
+						By("Delete htpasswd idp")
+						for _, idpName := range idpNames {
+							output, err = idpService.DeleteIDP(
+								clusterID,
+								idpName,
+							)
+							Expect(err).To(BeNil())
+							textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+							Expect(textData).Should(ContainSubstring("Successfully deleted identity provider '%s' from cluster '%s'", idpName, clusterID))
+						}
+
+					})
+
+				It("Test the scenario: create htpasswd idp with --from-file (htpasswdidp1)--> create htpasswd idp with (htpasswdidp3) --> create admin",
+					labels.Medium, labels.Runtime.Day2,
+					func() {
+
+						if profile.ClusterConfig.ExternalAuthConfig {
+							Skip("IDP is not supported for external auth")
+						}
```

Comment:
Could be moved in a `BeforeEach` inside the `Context`, so common to both scenarios

Reply:
done